katas/langtonant/adalid/lang.js

```diff
@@ -0,0 +1,98 @@
+const antMoves = (large = undefined, steps = undefined, sense = '',) =>{
```

Comment:
is this function used outside from this file? seems to have been deprecated by contents of `langtonant.js`, if so, this file should be deleted